src/data/STATUSES/layers/patch6.1.ts

```diff
@@ -5,6 +5,19 @@ import {SHARED} from '../root/SHARED'
 export const patch610: Layer<StatusRoot> = {
 	patch: '6.1',
 	data: {
+		// DRK 6.1 buff changes
+		BLOOD_WEAPON: {
+			duration: 15000,
+			stacksApplied: 5,
+		},
+
+		// Duration varies by the remaining duration on Walking Dead when it was cleansed
+		UNDEAD_REBIRTH: {
+			id: 3255,
+			name: 'Undeath Rebirth',
```

Comment:
Name doesn't match key. Which one matches the game data?

Reply:
I got bamboozled, the key is correct, updating the rest. Good catch.